src/server/src/main/java/io/cassandrareaper/resources/ClusterResource.java

```diff
@@ -267,14 +272,12 @@ private Cluster findClusterWithSeedHost(String seedHost, Optional<Integer> jmxPo
     String parsedClusterName = parseClusterNameFromSeedHost(seedHost).orElse("");
 
     try {
-      Cluster cluster
-          = new Cluster(
+      Cluster cluster = new Cluster(
               parsedClusterName,
               Optional.empty(),
               Sets.newHashSet(seedHost),
-              ClusterProperties.builder()
-                  .withJmxPort(jmxPort.orElse(Cluster.DEFAULT_JMX_PORT))
-                  .build());
+              ClusterProperties.builder().withJmxPort(jmxPort.orElse(Cluster.DEFAULT_JMX_PORT)) .build());
+
```

Comment:
Seems like you've been re-formatting code that hasn't been modified in this PR.
Makes the review more challenging ;)

Reply:
that's why there's two commits in the PR. review just the first commit ;)